crates/next-custom-transforms/tests/fixture/server-actions/modules.d.ts

```diff
@@ -49,3 +49,6 @@ declare module 'components' {
 declare module 'navigation' {
   export function redirect(href: string): void
 }
+
+// Some tests generate `data:text/javascript,...` imports
+declare module 'data:text/*'
```

Comment:
If only TS would allow template literal types in `declare module` to actually type arbitrary `data:text/javascript` 😄